### ISSUE TITLE
Suggestion: Remove the permanent whitelisting of simplewall itself--let the user decide

### ISSUE THREAD

**Issue by xEBFE** [henrypp/simplewall#106]
**simplewall 2.1.4**

Could you make simplewall more transparent about how it gives itself exemption from blocking by removing the following line in main.cpp:

```
2988			error_count += !_wfp_createrulefilter (nullptr, nullptr, app.GetBinaryPath (), DirBoth, nullptr, 0, AF_UNSPEC, false, FILTER_WEIGHT_APPLICATION, false);
```

Instead, could you just add simplewall's file executable as a default Allowed app the first time you install/run simplewall.

**Comment by henrypp** [comment]
Why? simplewall does not make any connect without user decision (update checking, dns resolution... except #94 bug, it's fixed, but not yet released ;).

**Comment by xEBFE** [comment]
I understand your point. However, I think it's a matter of just being transparent with users of simplewall. As there is no mention of itself being exempt from blocking. It wasn't until I added the simplewall process to the Blocked apps and tried updating and no notification popped up that I realized there was some self exemption going on. By adding simplewall (by default) to the Allowed apps you're letting them know that simplewall itself plays by it's own rules.
